ui/lib/core/addon/templates/components/replication-mode-summary.hbs

```diff
@@ -0,0 +1,101 @@
+{{#if (eq display 'menu')}}
+  <div class="level is-mobile">
+    <div class="level-left is-flex-1">
+      {{#if replicationUnsupported}}
+        Unsupported
+      {{else if replicationEnabled}}
+        <span>
+          {{capitalize modeForUrl}}
+        </span>
+        {{#if secondaryId}}
+          <small>
+            <code>
+              {{secondaryId}}
+            </code>
+          </small>
+        {{/if}}
+        <small>
+          <code>
+            {{clusterIdDisplay}}
+          </code>
+        </small>
+        {{else if (and (eq mode 'performance') (not (has-feature "Performance Replication")))}}
+        Learn more
+      {{else if auth.currentToken}}
+        Enable {{if (eq mode 'performance') 'Performance' 'DR'}}
+      {{else}}
+        Not enabled
+      {{/if}}
+    </div>
+    <div class="level-right">
+      {{#if replicationEnabled}}
+        {{#if (get cluster (concat mode 'StateGlyph'))}}
+          <span class="has-text-success">
+            {{i-con size=16 glyph=(get cluster (concat mode 'StateGlyph'))}}
+          </span>
+        {{else if syncProgress}}
+          <progress value="{{syncProgressPercent}}" max="100" class="progress is-small is-narrow is-info">
+            {{syncProgress.progress}} of {{syncProgress.total}} keys
+          </progress>
+        {{/if}}
+      {{else}}
+        <ICon @glyph="neutral-circled-outline" @size="16" @class="has-text-grey-light" />
```

Comment:
should this instance of `ICon` be commented out as well?

Reply:
Heh, probably - just a branch that's hard to hit locally.